deptofdefense/move.mil
# Add banners and logos for missing services

Addresses #371.

## Checklist

I have…


## Summary of Changes

This pull request adds banners and logos for the Marine Corps and Coast Guard to their respective service-specific information pages, and adds a logo for the Navy's page. 

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `remaining-service-banners`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in the Web browser of your choice.

## Screenshots

<details><summary>Marine Corps - Desktop Chrome</summary>

<img width="1027" alt="screen shot 2018-04-09 at 10 18 37 am" src="https://user-images.githubusercontent.com/26554826/38503371-e4592d8a-3bdf-11e8-9a45-148f59d1e019.png">

</details>

<details><summary>Marine Corps - iOS Safari</summary>

![img_b8a7f56262ad-1](https://user-images.githubusercontent.com/26554826/38503708-c0315f12-3be0-11e8-97b7-7ff17ed0db36.jpeg)

</details>

<details><summary>Marine Corps - Desktop IE11</summary>

![virtualbox_msedge - win10_09_04_2018_10_28_59](https://user-images.githubusercontent.com/26554826/38503776-eba83a80-3be0-11e8-9bba-e5cd02221b1d.png)

</details>

<details><summary>Navy - Desktop Chrome</summary>

<img width="1021" alt="screen shot 2018-04-09 at 10 18 45 am" src="https://user-images.githubusercontent.com/26554826/38503398-fc58d214-3bdf-11e8-8615-fb54c78fdd80.png">

</details>

<details><summary>Coast Guard - Desktop Chrome</summary>

<img width="1014" alt="screen shot 2018-04-09 at 10 18 52 am" src="https://user-images.githubusercontent.com/26554826/38503414-0318edfa-3be0-11e8-8ba9-4684ab0204c6.png">

</details>
